portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,11 +37,26 @@ public void init(){
         quotes.add("You can't have everything. Where would you put it?");
         quotes.add("Well-behaved women seldom make history.");
     }
+    public ArrayList messages(){
```

Comment:
Good use of a helper method to move the hardcoded message population away from the main `doGet()` logic!

nit: add space before opening curly brace.